### DISCUSSION TITLE
How to stitch a floor image?

### DISCUSSION THREAD

**Discussion by secretxu** [OpenStitching/stitching#141] [Q&A]
I have a series of photos of cement floors or walls, how can I stitch them into a complete plan？

<img width="1079" alt="1111" src="https://github.com/OpenStitching/stitching/assets/8953164/119df3d4-e83e-4be7-ac96-61aa9effacc4">

**Comment by lukasalexanderweber** [comment]
Start with two simple Images, e.g. x21 and x23. Use the verbose mode to understand how the stitching works. Incrementally add more images
